src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Net46.cs

```diff
@@ -0,0 +1,1193 @@
+        private ICredentials _defaultProxyCredentials;
+        private bool _preAuthenticate;
+        private bool _useDefaultCredentials;
+        private ICredentials _credentials;
+        private bool _allowAutoRedirect;
+        private int _maxAutomaticRedirections;
+        private string _connectionGroupName;
+        private ClientCertificateOption _clientCertOptions;
+        private X509Certificate2Collection _clientCertificates;
+        private IDictionary<String, Object> _properties;
+        private int _maxConnectionsPerServer;
+        private Func<
+            HttpRequestMessage,
+            X509Certificate2,
+            X509Chain,
+            SslPolicyErrors,
+            bool> _serverCertificateCustomValidationCallback;
+
+        #endregion Fields
+
+        #region Properties
+
+        public bool AllowAutoRedirect
+        {
+            get
+            {
+                return _allowAutoRedirect;
+            }
+
+            set
+            {
+                CheckDisposedOrStarted();
+                _allowAutoRedirect = value;
+            }
+        }
+
+        public DecompressionMethods AutomaticDecompression
+        {
+            get
+            {
+                return _automaticDecompression;
+            }
+
+            set
+            {
+                CheckDisposedOrStarted();
+                _automaticDecompression = value;
+            }
+        }
+
+        public bool CheckCertificateRevocationList
+        {
+            // TODO: Issue #11100.
+            get
+            {
+                throw new PlatformNotSupportedException();
+            }
+
+            set
+            {
+                CheckDisposedOrStarted();
+                throw new PlatformNotSupportedException();
+            }
+        }
+
+        public ClientCertificateOption ClientCertificateOptions
+        {
+            get
+            {
+                return _clientCertOptions;
+            }
+
+            set
+            {
+                if (value != ClientCertificateOption.Manual
+                    && value != ClientCertificateOption.Automatic)
+                {
+                    throw new ArgumentOutOfRangeException("value");
+                }
+
+                CheckDisposedOrStarted();
+                _clientCertOptions = value;
+            }
+        }
+
+        public X509CertificateCollection ClientCertificates
+        {
+            get
+            {
+                if (_clientCertificates == null)
+                {
+                    _clientCertificates = new X509Certificate2Collection();
+                }
+
+                return _clientCertificates;
+            }
+        }
+
+        public CookieContainer CookieContainer
+        {
+            get
+            {
+                return _cookieContainer;
+            }
+
+            set
+            {
+                if (value == null)
+                {
+                    throw new ArgumentNullException("value");
```

Comment:
Can we use nameof here and elsewhere it's relevant?  I believe we had already transformed the codebase with it by the 1.1 release.  If I'm wrong, it's fine as-is.